blueocean-dashboard/src/main/js/components/branchSource/services/BranchService.js

```diff
@@ -0,0 +1,38 @@
+import { BunkerService } from '@jenkins-cd/blueocean-core-js';
+import { BranchPager } from './BranchPager';
+/*
+ * This class provides a simple branch service.
+ *
+ * @export
+ * @class BranchService
+ * @extends {BunkerService}
+ */
+export class BranchService extends BunkerService {
+    /**
+     * Gets the Branch pager
+     *
+     * @param {string} url we are requesting
+     * @returns {Pager} Pager for this pipelne.
+     */
+    branchPager(url) {
+        return this.pagerService.getPager({
```

Comment:
Should not be using pager service for this. We cache pagers because we need the ordered list saved. This just needs one branch so we should not need to cache it.